Formula/emacs-plus@28.rb

```diff
@@ -82,7 +83,8 @@ class EmacsPlusAT28 < EmacsBase
   # Patches
   #
 
-  local_patch "no-titlebar", sha: "2fa80efc5cda7e96d88a5d145c9313092a6e53d38825c41967c745f08778c41b" if build.with? "no-titlebar"
+  local_patch "no-titlebar", sha: "69c246224fa7dc4bcb84a8286e8f67eb519e2699319010054a9f71a301a9e3bd" if build.with? "no-titlebar"
```

Comment:
I think this change is also made by mistake, this patch wasn't touched, so SHA should remain the same.